firmware/aek/aek.h

```diff
@@ -30,7 +30,7 @@
 #define ____ KC_NO
 
 #define LAYOUT( \
-    K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, \
+    K000,    K001, K002, K003, K004,   K005, K006, K007, K008,   K009, \
```

Comment:
Bunch of extra spaces?

Reply:
It was the start of cleaning it up to better represent the layour, but I ended up going a different route

Reply:
removed extra spaces